tests/Unit/Evolution/Systems/GrMhd/ValenciaDivClean/TestFunctions.py

```diff
@@ -0,0 +1,62 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+
+# Functions for testing Fluxes.cpp
+def tilde_d_flux(tilde_d, tilde_tau, tilde_s, tilde_b, tilde_phi, lapse, shift,
+                 sqrt_det_spatial_metric, spatial_metric, inv_spatial_metric,
+                 pressure, spatial_velocity, lorentz_factor, magnetic_field):
+    return tilde_d * (lapse * spatial_velocity - shift)
+
+
+def tilde_tau_flux(tilde_d, tilde_tau, tilde_s, tilde_b, tilde_phi, lapse, shift,
+                   sqrt_det_spatial_metric, spatial_metric, inv_spatial_metric,
+                   pressure, spatial_velocity, lorentz_factor, magnetic_field):
+    bsq = np.einsum("ab, ab", spatial_metric,
+                    np.outer(magnetic_field, magnetic_field))
+    b_dot_v = np.einsum("ab, ab", spatial_metric,
+                        np.outer(magnetic_field, spatial_velocity))
+    p_star = pressure + 0.5*(b_dot_v**2 + bsq/lorentz_factor**2)
```

Comment:
This is something you can take or leave, since I think refactoring too much will make the functions harder to read, but since you the expressions for `p_star` etc. are the same in different functions, might it be worth refactoring some into their own functions?